pkg/alertmanager/operator.go

```diff
@@ -150,6 +151,12 @@ func (c *Operator) RegisterMetrics(r prometheus.Registerer) {
 		Help: "Number of errors that occurred while reconciling the alertmanager statefulset",
 	})
 
+	c.triggerByCounterVec = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "prometheus_operator_triggered_total",
+		Help: "Number of times a Kubernetes object add, delete or update event" +
+			" triggered Prometheus Operator to reconcile an object",
```

Comment:
tiny nit: could we make this a full sentence and add a `the` between `triggered` and `Prometheus Operator` so that it reads more cleanly?